core/src/test/java/kafka/log/remote/RemoteLogManagerTest.java

```diff
@@ -913,6 +969,12 @@ private RemoteLogManagerConfig createRLMConfig(Properties props) {
         props.put(RemoteLogManagerConfig.REMOTE_LOG_STORAGE_SYSTEM_ENABLE_PROP, true);
         props.put(RemoteLogManagerConfig.REMOTE_STORAGE_MANAGER_CLASS_NAME_PROP, NoOpRemoteStorageManager.class.getName());
         props.put(RemoteLogManagerConfig.REMOTE_LOG_METADATA_MANAGER_CLASS_NAME_PROP, NoOpRemoteLogMetadataManager.class.getName());
+        props.put(REMOTE_LOG_METADATA_TOPIC_PARTITIONS_PROP, remoteLogMetadataTopicPartitionsNum);
+        props.put(remoteLogMetadataTestProp, remoteLogMetadataTestVal);
+        props.put(remoteLogMetadataCommonClientTestProp, remoteLogMetadataCommonClientTestVal);
+        props.put(remoteLogMetadataConsumerTestProp, remoteLogMetadataConsumerTestVal);
+        props.put(remoteLogMetadataProducerTestProp, remoteLogMetadataProducerTestVal);
```

Comment:
Perhaps, create a `TestUtils.remoteLogTopicBasedRlmmConfig()` similar to `TestUtils.producerConfig` or `TestUtils.createBrokerConfigs`

Reply:
I understand what you meant, I tried but that will bring the dependency with Storage module into client module, which is unexpected. I'm keeping it as is. Thanks.